app/main.py

```diff
@@ -4,7 +4,7 @@
 
 from app.core.config import config
 from app.core.security import security
-from app.ents.base.crud import db
+from app.ents.base.crud import db, migrate
```

Comment:
It looked like you do not use `migrate` in app/ents/base/crud.py.
Hence, why don't you initialize it here in app/main.py instead of there and later import it?

Reply:
Of right! It is because most times, migrate works with db, i thought I should initialize ``migrate`` where ``db`` was initialized. But regardless, I could just change it.